sn_interface/src/messaging/network_knowledge.rs

```diff
@@ -103,6 +103,6 @@ pub struct SectionsDAG {
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
 pub struct SectionTreeUpdate {
     pub section_auth: SectionAuthorityProvider,
-    pub section_signed: KeyedSig,
+    pub section_signed: SectionSig,
```

Comment:
`section_signed` is confusing now that we have a `SectionSigned` struct, let's rename to `section_sig`
```suggestion
    pub section_sig: SectionSig,
```

Reply:
agreed!